src/components/activity/ActivityDirectivesTablePanel.svelte

```diff
@@ -101,8 +102,37 @@
       hide: true,
       resizable: true,
       sortable: false,
+      autoHeight: true,
+      cellRenderer: (params: ICellRendererParams<ActivityDirective>) => {
+        const div = document.createElement('div');
```

Comment:
I would suggest making a svelte component for this. You can use the `ActivityDirectivesTable` component's usage of the `DataGridActions` component as a reference. You will have a much more intuitive control over the rendering. You can then move the ordering/formatting logic into that new component as well. Then you won't need the `valueGetter` definition in this column anymore since that'll go into the new component.